wetterdienst/provider/nws/observation/api.py

```diff
@@ -20,7 +20,7 @@
 from wetterdienst.util.parameter import DatasetTreeCore
 
 Settings.fsspec_client_kwargs["headers"] = {
-    "User-Agent": "wetterdienst/gutzemann@gmail.com",
+    "User-Agent": "wetterdienst/0.48.0",
```

Comment:
Until we can use the real version number at runtime, I think it will be better than nothing.

Reply:
The reason I put my mail there is their hint 
"""
Authentication
A User Agent is required to identify your application. This string can be anything, and the more unique to your application the less likely it will be affected by a security event. If you include contact information (website or email), we can contact you if your string is associated to a security event. This will be replaced with an API key in the future.
User-Agent: (myweatherapp.com, contact@myweatherapp.com)
"""